Userland/Applications/Browser/BrowserWindow.cpp

```diff
@@ -615,6 +615,10 @@ void BrowserWindow::create_new_tab(URL url, bool activate)
         return active_tab().view().get_element_tag_name(element_id);
     };
 
+    new_tab.webdriver_endpoints().on_take_screenshot = [this]() {
```

Comment:
This one seems useful from a non-WebDriver viewpoint as well, I'd make that a regular tab callback